tldr/semantic_old.py

```
@@ -0,0 +1,1062 @@
+            if not target_languages:
+                return 0
+            units = []
+            for lang_name in target_languages:
+                units.extend(extract_units_from_project(str(project), lang=lang_name, respect_ignore=respect_ignore))
+        else:
+            units = extract_units_from_project(str(project), lang=lang, respect_ignore=respect_ignore)
+
+    if not units:
+        return 0
+
+    import numpy as np
```

Ruff (0.14.10)

Redefinition of unused `np` from line 861

Remove definition: `np`

(F811)